rubrics/api/src/main/bundle/rubrics.properties

```diff
@@ -62,3 +62,13 @@ draft_evaluation=Draft rubric evaluation. Save in the tool to publish the rubric
 locked_message=This tool, when locked (the default), can NOT be seen by students. Students can \
 still, however, see their rubric evaluations in tools like Assignments, Gradebook and Tests and \
 Quizzes. You DO NOT need to unlock this tool for students to see their rubric evaluations.
+
+# Rubrics in pdf
+export_rubric_title=Exported Rubric Title: 
```

Comment:
Do not use these patterns for strings that will have values, use arguments:

Example: https://github.com/sakaiproject/sakai/blob/master/master/pom.xml#L48

Example: export_rubric_title=Exported Rubric Title: {}

Same for all the strings, use the prefix export_ for all of these strings.

Reply:
Done.